### ISSUE TITLE
Prophecies Support

### ISSUE THREAD

**Issue by nicksloan11** [thirdy/POE-TradeMacro#33]
Would be nice to be able check prices on Sealing Prophecies when you mouse over the prophecies on your prophecies map.

It lets you do this now, but it just searches for the last search you ran.

**Comment by Eruyome** [comment]
That's not gonna happen. I thought about this, too. `ctrl+c` copies the items data to the clipboard when you hover over an item, this doesn't happen when you hover over the prophecies on the prophecies map. Therefore we don't have any information about the prophecy, we don't even know that there is one.

**Comment by Penetrosity** [comment]
In cases such as these the user can just use ctrl-i to type in the prophecy themselves.

**Comment by Eruyome** [comment]
That's true, but way more complicated than speaking to Navali -> Seal Prophecy. Isn't it? I mean, who wants to type the prohecies name?

**Comment by Eruyome** [comment]
Just an FYI, I asked Chris if they could add support for this to the game (ctrl + c copying prophecy data into the clipboard). He wanted to discuss this but there's no guarantee that we will ever get it.
